lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -259,6 +259,19 @@ static bool recursiveType(const StructType *ST, const Type *Ty) {
   return Run(Ty);
 }
 
+static bool isManifestConstant(const Constant *c) {
```

Comment:
Hi, @AlexeySachkov 
Based the pre-ci log, we have some tidying errors here, could you help replace the input parameter "const Constant  *c" with "const Constant *C"?
Thanks very much.